weaveworks/tf-controller
# implement break the glass mechanism

Fixes #617 
Fixes #625

- [x] Need to address this: #625 too

How to test:

- `./tools/reboot.sh`
- `make all`
- `tilt up`
- `bin/tfctl break-glass helloworld-tf`
and it should get into a shell.
in the shell try: `terraform output` and you should see an output.
